### PR TITLE
fixes error where errors detected by metapool aren't reported.

### PR DESCRIPTION
**Pull request by charles-cowart**

### CODE REVIEW

**Comment by charles-cowart** [comment]
Hi @wasade, since @antgonza is out, would you mind reviewing this second hot fix for me please? Thank you so much!

File: sequence_processing_pipeline/Pipeline.py [hunk 1]
```diff
@@ -240,13 +240,17 @@ def _validate_sample_sheet(self, sample_sheet_path):
...
-                                    '\n'.join([str(x) for x in msgs]))
+                                    '\n' + '\n'.join([str(x) for x in msgs]))
 
             # return a valid sample-sheet, and preserve any warning
             # messages
             self.warnings += [str(x) for x in msgs if
                               isinstance(x, WarningMessage)]
             return val_sheet
+        else:
+            # if val_sheet is not None, assume msgs contains Errors.
```
Comment (wasade) [inline sequence_processing_pipeline/Pipeline.py:251]:
```suggestion
            # if val_sheet is None, assume msgs contains Errors.
```

**Review by wasade — COMMENTED** [review]
The change suggests that `_validate_sample_sheet` could previously return `None`. Does anything need to change with the caller?

File: sequence_processing_pipeline/Pipeline.py (hunk 1 above)
Comment (charles-cowart) [reply sequence_processing_pipeline/Pipeline.py]: nice catch! :)

**Comment by charles-cowart** [comment]
@wasade Nope. I believe before we added the amplicon pathway, we raised an Error if val_sheet was None (or the caller raised an Error if None). That has appeared to have gone away and was what caused the error Gail encountered this morning. This fix will raise an Error that will report the errors found back to the UI so Gail and the others can view them as before.

**Comment by charles-cowart** [comment]
closing w/out merging as this fix was superseded by https://github.com/biocore/mg-scripts/pull/96

**Pull request closed without merging**